demo/admin/crud-generator-config.ts

```diff
@@ -0,0 +1,11 @@
+import { CrudGeneratorConfig } from "@comet/cms-admin";
+export default [
```

Comment:
Have you thought about instead of having a config file to extract this information from the schema as well? The `@CrudGenerator()`-Decorator could add a GraphQL comment or directive to show that the code was generated in the API and should probably be generated in the Admin as well? The `target` could be calculated using the object type name.

Reply:
while it would be really nice to have no config file, but there is no real relation to the api generator:

- it works also for completely hand made api (at least for the parts that are compatible)
- more importantly: if you use scaffolding on api side, the `@CrudGenerator` is (currently) removed

alternative idea: *if* we would use admin generator as scaffolding only tool, we could promp for which entity to generate.

But my plan is still, that we should be able to use generation in standard cases. We will see if those cases really exist.